lib/readline.js

```diff
@@ -1331,7 +1331,8 @@ function codePointAt(str, index) {
   }
   return code;
 }
-exports.codePointAt = codePointAt;
+exports.codePointAt = util.deprecate(codePointAt,
+    'codePointAt() is deprecated. Use String.prototype.codePointAt');
```

Comment:
FWIW, I agree with @vkurchatkin.  I believe the reason it was exported was so it could be unit-tested in the test suite.